source/quick-start.txt

```diff
@@ -2,5 +2,173 @@
 Quick Start
 ===========
 
-
 .. default-domain:: mongodb
+
+Check for Node and NPM
+----------------------
+
+Ensure your system has Node version 8 or later and NPM (Node Package
+Manager) installed. Run the following commands to check thatr you meet
+these requirements:
+
+First, check the version of Node
+
+.. code-block:: sh
+
+   node -v
+
+You should see output similar to the following:
+
+.. code-block:: sh
+
+   v12.14.1
+
+Next, check the version of NPM
+
+.. code-block:: sh
+
+   npm -v
+
+You should see output similar to the following:
+
+.. code-block:: sh
+
+   6.13.4
+
+If you do not see any output or encounter an error, you will need to
+install Node and NPM. Visit `nodejs.org <https://nodejs.org>`_ for
+installation instructions.
+
+Initialize the Project
+----------------------
+
+Navigate to a new directory and initialize a new project:
```

Comment:
Nitpick: this should probably identify that they need to run three commands as it might not be clear to someone relatively new to a shell.

> Navigate to a new directory and initialize a new project with the following three commands:

Reply:
Splitting these commands out instead to make it easier for a newcomer